tutorials/R/R-skills/using-r-markdown/Rmd02-RMarkdown/Rmd02-RMarkdown.Rmd

```diff
@@ -255,26 +260,53 @@ name as it is required for more advanced knitting approaches.
 
 Continue working on your document. Below the last section that you've just added, 
 create a code chunk that loads the packages required to work with raster data 
-in R, and sets the working directory.
+in R.
 
-<pre><code>```{r setup-read-data }
+<pre><code>```{r setup-library }
+   
    library(rgdal)
    library(raster)
+ 
+ ```</code></pre>
```

Comment:
Excellent addition here! This will be super helpful once I finally make a new version of the set working directory tutorial